practice-two/src/constants/regexp.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-useless-escape */
+export const REGEX = {
+  USERNAME: '^[A-Za-z0-9]{3,16}$',
+  AVATAR: '\.(jpg|jpeg|png|gif|bmp|tiff|webp|svg|ico)$',
+  EMAIL: '^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$',
+  PHONENUMBER: '^(((\+|)84)|0)([1-9]{1})([0-9]{8})\b',
+  DESCRIPTION: '^.{3,20}$',
+  ADDRESS: '.{8,}',
+};
+ 
```

Comment:
add space line

Reply:
I rechecked that I added space line before pushing code onto git
![image](https://github.com/datdragon777/React-Training/assets/74806826/a5fd3910-aa1c-43ac-afd6-44b2c235b34a)